### PR TITLE
Home: added area selector

### PR DESCRIPTION
**Pull request by minotogna**
WIP #396

### CODE REVIEW

File: webapp/app/landing/Introduction/AreaSelector/AreaSelector.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import axios from 'axios'
+
+import { Area, Country } from '@common/country'
+import * as BasePaths from '@webapp/main/basePaths'
+
+import { useI18n } from '@webapp/components/hooks'
+
+import DropdownAreas from './DropdownAreas'
+
+const areas = {
```
Comment (sorja) [inline webapp/app/landing/Introduction/AreaSelector/AreaSelector.js:13]: Should this be in common/country/area?

File: webapp/app/landing/Introduction/AreaSelector/AreaSelector.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+const AreaSelector = () => {
+  const i18n = useI18n()
+  const [dropdownOpened, setDropdownOpened] = useState('')
+  const [countryISOs, setCountryISOs] = useState([])
+
+  // on mount fetch countries
+  useEffect(() => {
+    ;(async () => {
+      const { data } = await axios.get(`/api/countries`)
```
Comment (sorja) [inline webapp/app/landing/Introduction/AreaSelector/AreaSelector.js:26]: use hook useGetRequest

File: webapp/app/landing/Introduction/AreaSelector/AreaSelector.js (hunk 1 above)
Comment (minotogna) [reply webapp/app/landing/Introduction/AreaSelector/AreaSelector.js:13]: This is local state . The keys of the dropdown

File: webapp/app/landing/Introduction/AreaSelector/AreaSelector.js (hunk 2 above)
Comment (minotogna) [reply webapp/app/landing/Introduction/AreaSelector/AreaSelector.js:26]: I wanted to avoid double setState .

**Pull request merged**